FileAPI/file/send-file-form-windows-1252.tentative.html

```diff
@@ -33,15 +33,14 @@
   fileNameSource: 'x-user-defined',
   fileBaseName: 'file-for-upload-in-form-\uF7F0\uF793\uF783\uF7A0.txt',
   formEncoding: 'windows-1252',
-  expectedEncodedBaseName: (
-      'file-for-upload-in-form-&#63472;&#63379;&#63363;&#63392;.txt'),
+  expectedEncodedBaseName: 'file-for-upload-in-form-&#63472;&#63379;&#63363;&#63392;.txt',
```

Comment:
Why does this not use the \x escapes like the others?

Reply:
The \x escapes represent a control or non-ASCII byte that reached the server. The charref escapes are escaped when building the multipart/form-data payload. I should probably add a comment to make that clear.

Reply:
Oh I see, because these code points cannot be encoded in that encoding.